core/config/config.go

```diff
@@ -65,6 +65,10 @@ func Load(ctx context.Context, db pg.DB, sdb *sinkdb.DB) (*Config, error) {
 	if err != nil {
 		return nil, errors.Wrap(err)
 	} else if ver.Exists() {
+		raftID := c.Id
+		if !idMatchesPG(ctx, raftID, db) {
+			panic("Raft core ID doesn't match Postgres core ID: do you have a stale Raft config? Try: `rm -rf ~/.chaincore/raft`")
```

Comment:
Don't hardcode `~/.chaincore/raft` here, populate it with the actual value, lest you advise people to remove the wrong directory (e.g. while testing with an alternate config).

Don't use backticks, which aren't meaningful in the console or log file where this message is likely to show up.

I might even consider rewording this message so that it doesn't contain a copy-and-paste command. For most things that would be a useful convenience, but for an `rm -rf` it's not a bad idea to require a little extra user intervention.